mcp-servers/Dockerfile.snowflake

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/astral-sh/uv:python3.12-bookworm-slim AS builder
+FROM ghcr.io/astral-sh/uv:python3.12-trixie AS builder
```

Comment:
Any reason you didn't use the `slim` version here? Also, is something stopping us from using python 3.14?

Reply:
Updated it to use slim in a separate commit. Was sticking with Python 3.12 for now since the original Snowflake Dockerfile uses that version. 3.14 should work as well.